modules/pam_namespace/pam_namespace.c

```diff
@@ -34,6 +34,8 @@
 
 #define _ATFILE_SOURCE
 
+#include <linux/close_range.h>
+
```

Comment:
I doubt we can assume that this header file is universally available,
but we might not need it after all, see below.